backend/controllers/controller.ts

```diff
@@ -105,7 +108,24 @@ export const factory = ({
 
   const getCaseNote = asyncMiddleware(async (req, res) => {
     const { offenderNumber, caseNoteId } = req.params
-    const caseNote = await caseNotesApi.getCaseNote(res.locals, offenderNumber, caseNoteId)
+    const { context } = await getContextWithClientTokenAndRoles({
+      offenderNo: offenderNumber,
+      res,
+      req,
+      oauthApi: null,
+      systemOauthClient: null,
+      restrictedPatientApi: null,
+    })
+    try {
+      const prisonerDetails = await prisonApi.getDetails(res.locals, offenderNumber)
+      contextProperties.setCustomRequestHeaders(context, { CaseloadId: prisonerDetails.agencyId })
+    } catch (e) {
```

Comment:
What could fail here that would cause this exception? Could do prisonerDetails?.agencyId to get rid of the try catch perhaps?

Reply:
i added the try catch because i was afraid of potential exception from `prisonApi.getDetails`
but then the int tests are passing now so it should be okay to go without this try catch.